packages/cli/src/commands/doctor/healthchecks/iosDeploy.ts

```diff
@@ -1,58 +1,117 @@
 import execa from 'execa';
+import chalk from 'chalk';
+// @ts-ignore untyped
+import inquirer from 'inquirer';
+import {logger} from '@react-native-community/cli-tools';
 import {checkSoftwareInstalled, PACKAGE_MANAGERS} from '../checkInstallation';
 import {packageManager} from './packageManagers';
-import {logManualInstallation} from './common';
+import {logManualInstallation, removeMessage} from './common';
 import {HealthCheckInterface} from '../types';
+import {Ora} from 'ora';
 
-const getInstallationCommand = () => {
+const label = 'ios-deploy';
+
+const installationWithYarn = 'yarn global add ios-deploy';
+const installationWithNpm = 'npm install ios-deploy --global';
+
+const identifyInstallationCommand = () => {
   if (packageManager === PACKAGE_MANAGERS.YARN) {
-    return 'yarn global add ios-deploy';
+    return installationWithYarn;
   }
 
   if (packageManager === PACKAGE_MANAGERS.NPM) {
-    return 'npm install ios-deploy --global';
+    return installationWithNpm;
   }
 
   return undefined;
 };
 
+const installLibrary = async ({
+  installationCommand,
+  packageManagerToUse,
+  loader,
+}: {
+  installationCommand: string;
+  packageManagerToUse: 'yarn' | 'npm';
+  loader: Ora;
+}) => {
+  try {
+    loader.start(`${label} (installing with ${packageManagerToUse})`);
+
+    await execa(installationCommand);
+
+    loader.succeed(`${label} (installed with ${packageManagerToUse})`);
+  } catch (error) {
+    loader.fail();
+    logger.log(chalk.dim(`\n${error}`));
```

Comment:
Shuoldn't we just log the message? We don't need the stack trace here
```suggestion
    logger.log(chalk.dim(`\n${error.message}`));
```

Reply:
I checked with `error` and it returns the same as `error.message`, I'll change this though.

In `cocoapods` we are also logging only `error`, shall I change that there as well?

Reply:
Please do

Reply:
Done!